### PR TITLE
Add Tests and Packaging Instructions

### LINKED ISSUE #1: Create setup.py for pynaco
We need to have a setup.py and manifest.in for pynaco.

### PR DESCRIPTION
**Pull request by damonkelley**
Will resolve #1 

The tests added in this branch use the the files in `tests/data` to get expected input and output and repeatedly pass them to `test_normalize` and `test_normalizeSimplified`. To do this, we use the ddt package.

### CODE REVIEW

File: tests/naco_test.py [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+    return standard
 
 
+def simplified_data():
+    """Data provider method for the normalizeSimplified function."""
+    script = open(os.path.join(DATA_PATH, 'NACO.script'))
+    check = open(os.path.join(DATA_PATH, 'NACOsimplified.check'))
+
+    # Create a list of tuples in the form of
+    # (unormalized data, expected output)
```
Comment (damonkelley) [inline tests/naco_test.py]: I think this is a typo. Should be 'unnormalized'.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,9 @@
 *.pyc
+*.egg
+*.swp
+*.swo
+
+pynaco.egg-info
+dist/
+build/
+
```
Comment (damonkelley) [inline .gitignore]: Remove the newline at EOF

File: runtests.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env python
```
Comment (damonkelley) [inline runtests.py]: Get rid of the space after the shebang

**Comment by damonkelley** [comment]
Ping @ldko @vphill 

Mind giving these changes a once over?

**Comment by ldko** [comment]
Once over given.

**Pull request merged**